webpush_test.go

```diff
@@ -76,3 +77,17 @@ func TestSendNotificationToStandardEncodedSubscription(t *testing.T) {
 		)
 	}
 }
+
+func TestSendTooLargeNotification(t *testing.T) {
+	_, err := SendNotification([]byte(strings.Repeat("Test", int(MaxRecordSize))), getStandardEncodedTestSubscription(), &Options{
+		HTTPClient:      &testHTTPClient{},
+		Subscriber:      "<EMAIL@EXAMPLE.COM>",
+		Topic:           "test_topic",
+		TTL:             0,
+		Urgency:         "low",
+		VAPIDPrivateKey: "testKey",
+	})
+	if err == nil {
+		t.Fatal("Expected error with too large payload")
```

Comment:
Could you format this message like the other `t.Fatal` messages above using the `ErrMaxPadExceeded` error? (Might also make sense to check if it is the correct error not just `nil`)

```go
t.Fatalf("Error is nil, expected=%s, ErrMaxPadExceeded)
```